src/compiler/parser.ts

```diff
@@ -6715,7 +6706,11 @@ namespace ts {
                     result.postParameterName = postName;
                     result.name = postName || preName;
                     result.isBracketed = isBracketed;
-                    return finishNode(result);
+                    // Include trailing whitespace inside the range of the `@param` tag.
+                    // This way we can provide completions after `@param `.
+                    let end = scanner.getStartPos();
+                    while (ts.isWhiteSpaceSingleLine(scanner.getText().charCodeAt(end))) end++;

```

Comment:
I was using it so that at `@param    /*completions here*/` you would get completions by being inside the `@param` node. But we can do this a different way by changing `getJsDocTagAtPosition` to handle ranges differently.